maturity-model.md

```diff
@@ -0,0 +1,175 @@
+* DP-3: Most teams develop their own CI process, using corporate CI tools. There are CI environments. Code review process defined, and used by some projects. Code review is usually done by external team members.
+
+**Decisions**
+
+* DC-0: Decision-makers often withhold data and resources without explanation.
+* DC-1: Materials that are part of decision-making practices become available for review after decisions are finalized.
+* DC-2:	People feel like they know about—and are helping to shape—most (but not all) important decisions as those decisions are unfolding. Materials that are part of decision-making practices are available at defined project milestones
+* DC-3: People feel like they are a part of a shared, standard process for collective decision-making that the organization endorses. Materials that are part of decision-making practices are continuously accessible during work processes.
+
+**Helpful Resources**
+
+* RS-0: Individuals and teams neither contribute to nor draw upon a shared repository of knowledge.
+* RS-1:	Individuals and teams release project materials for review internally, after they've finished their work. Individuals and teams share resources, but in disconnected, fragmented, or individualized/siloed systems or repositories. Individuals and teams share resources, but there is no commonly expressed or shared understanding of the criteria used to determine whether information is sensitive or not. Do not "share thinking on others".
+* RS-2: Individuals and teams make project-related materials accessible to some members of project teams according to clearly defined and shared formats and/or protocols. Individuals and teams withhold sensitive data and resources, provide limited details, context, and scope.
+* RS-3: Individuals and teams make project-related materials broadly accessible to the organization—and possibly outside the organization as well—according to clearly defined and shared formats and/or protocols. Individuals and teams who must withold sensitive data and resources are clear about what they're not sharing, and others understand why those materials are not available to them.
+
+**Stories**
+
+* ST-0: Individuals and teams don't share neither successes nor failures for others to learn from.
+* ST-1: Individuals and teams are comfortable sharing stories about successes, but not about failures.
+* ST-23	Individuals and teams are comfortable sharing stories of successes and failures during retrospectives and reviews.
+* ST-3:	Individuals and teams are comfortable sharing stories of successes and failures, and learn from failures according to formal protocols.
+
+### Collaboration
+
+**Channels for Providing Feedback**
+
```

Comment:
This point seems to mix several aspects into one.

Likely it tries to talk about "Communication channels for projects", "Feedback culture", "Channels for development discussions", "Channels for user support"?

Reply:
Initially any channel of communication that helps to start building a community, or several of them, within the organization.

Thus, there final goal is to have proper communication channels known by anyone in the organization and potentially with different goals (announcements, training, project channels, infra discussions, etc).

The goal here is not to define the really specific communication channels you need, but make it clear that you need them and that they should follow certain guidelines (e.g., netiquette). This again helps to have an homogeneous way of behaving within the organization (e.g., using a specific provider for this), following certain rules (e.g., create a project channel each time there's a new project, open the channel to the public, foster discussions in the channels), and with the goal of having a referenceable and archivable body of knowledge (e.g., as in the case of the ASF with mailing list discussions).

Reply:
Likely this goal should then be added as further explanation of the point :)

Reply:
I tried to rephrase and add as an introduction sentence.